doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -96,14 +97,17 @@ The verification provided by this bootloader is recommended and suitable for all
 * Metadata verification - Checks that the images are compatible.
 
 .. caution::
-   You must generate and use your own signing keys while in development and before deploying when using either MCUboot or the |NSIB| as an immutable bootloader.
-   See :ref:`ug_fw_update_development_keys` for more information.
+   * You must generate and use your own signing keys while in development and before deploying when using either MCUboot or the |NSIB| as an immutable bootloader.
```

Comment:
btw. This part contains generic information about bootloaders in the NCS. Maybe we can make it part of a generic introduction ?